packages/web-components/src/components/cbp-card/cbp-card.tsx

```diff
@@ -70,6 +70,7 @@ export class CbpCard {
                 {(this.variant !== 'banner') && <slot name="cbp-card-title" />}
                 <slot />
             </div>
+            <slot name='cbp-card-actions' />
```

Comment:
I don't see this filtering out interactive cards. I think {!this.interactive &&  <slot name='cbp-card-actions' />} is needed

Reply:
updated